src/main/java/org/craftercms/studio/impl/v2/service/content/ContentServiceImpl.java

```diff
@@ -331,6 +336,17 @@ public Resource getContentAsResource(@ValidateStringParam String site,
         return contentServiceV1.getContentAsResource(site, path);
     }
 
+    @Override
+    @Valid
+    @ValidateAction(type = Type.CREATE)
+    @HasPermission(type = DefaultPermission.class, action = PERMISSION_CONTENT_WRITE)
```

Comment:
What about `folder_create` permission?

Reply:
@phuongnq let's use `folder_create`.
Let's also rename it to be `create_folder` so it is consistent with the permission names we have been using

Reply:
Updated to use `create_folder`. Thank you!
Note: We may need to review the PR when merging this to make sure the pipeline is up-to-date with other PRs which modify the same.